--- ./configure.in.org	Wed Jul  3 17:06:25 2002
+++ ./configure.in	Wed Jul  3 17:41:11 2002
@@ -61,47 +61,48 @@
 
 dnl  Math library
 AC_CHECK_HEADERS(math.h,
-	AC_CHECK_LIB(m,pow,,AC_MSG_WARN([Cannot find Math library.])),
-	AC_MSG_WARN([Cannot find math.h.]))
+		 [AC_CHECK_LIB( m, pow, [], AC_MSG_WARN([Cannot find Math library.]) )],
+		 [AC_MSG_WARN([Cannot find math.h.])])
 
 dnl  Memory debugging
 AC_ARG_ENABLE(memdebug,
 	[  --enable-memdebug       enable build-in memory debugging],
-	if `test "$enable_memdebug" != "no"`
+	[if `test "$enable_memdebug" != "no"`
 	then
 	{
-		AC_CHECK_HEADER(pthread.h,[
-			AC_CHECK_LIB(pthread, pthread_mutex_init,
-				[AC_DEFINE(HAVE_MEM_DEBUG)
-				LIBS="$LIBS -lpthread"
-				AC_MSG_RESULT("turning memory debugging... on")],
-				AC_MSG_WARN(pthread library not present),)],
-			[AC_MSG_WARN(pthread library not present)
-			AC_MSG_RESULT("turning memory debugging... off")])
+		AC_CHECK_HEADER(pthread.h,[ AC_CHECK_LIB(pthread, pthread_mutex_init,
+					   [ AC_DEFINE(HAVE_MEM_DEBUG)
+					   LIBS="$LIBS -lpthread"
+					   AC_MSG_RESULT(["turning memory debugging... on"])
+					   ],	AC_MSG_WARN([pthread library not present])) 
+					  ],
+					  [AC_MSG_WARN(pthread library not present)
+					  AC_MSG_RESULT(["turning memory debugging... off"])]
+				)
 	}
 	else
-		AC_MSG_RESULT("turning memory debugging... off")
-	fi
-	, AC_MSG_RESULT("turning memory debugging... off")
+		AC_MSG_RESULT(["turning memory debugging... off"])
+	fi]
+	,[ AC_MSG_RESULT(["turning memory debugging... off"]) ]
 	)
 
 dnl  Graphic debugging
 AC_ARG_ENABLE(gfxdebug,
      [  --enable-gfxdebug	  enable build-in graphic debugging (ONLY FOR TESTING PURPOSE)],
-	if `test "$enable_gfxdebug" != "no"`
-	then
-	{
-		AC_CHECK_HEADERS(X11/Xlib.h,
-			AC_CHECK_LIB(X11,XDrawLine,,AC_MSG_ERROR(X11 library not present)),
-			AC_MSG_ERROR(X11 library not present))
-		LDFLAGS="$LDFLAGS -L/usr/X11R6/lib"
-		AC_DEFINE(HAVE_GFX_DEBUG)
-		printf "turning graphic debugging... on\n"
-	}
+	[if `test "$enable_gfxdebug" != "no"`
+	 then
+		 {
+			 AC_CHECK_HEADERS(X11/Xlib.h,
+					  AC_CHECK_LIB(X11,XDrawLine,,AC_MSG_ERROR(X11 library not present)),
+					  AC_MSG_ERROR(X11 library not present))
+			 LDFLAGS="$LDFLAGS -L/usr/X11R6/lib"
+			 AC_DEFINE(HAVE_GFX_DEBUG)
+			 AC_MSG_RESULT(["turning graphic debugging... on"])
+		 }
 	else
-		printf "turning graphic debugging... off\n"
-	fi
-	, printf "turning graphic debugging... off\n"
+		AC_MSG_RESULT(["turning graphic debugging... off"])
+	fi]
+	,[ AC_MSG_RESULT(["turning graphic debugging... off"]) ]
 	)
 
 case "$target" in
@@ -113,8 +114,7 @@
 			;;
 esac
 
-AC_OUTPUT( 	        \
-	src/Makefile    \
-	src/examples/Makefile \
-	doc/Makefile \
-	Makefile)
+AC_OUTPUT([src/Makefile \
+	   src/examples/Makefile \
+	   doc/Makefile \
+	   Makefile])
